listeners.py

```diff
@@ -178,6 +179,86 @@ def __del__(self):
                 pass
 
 
+class StatisticsMonitor(BaseWatcher):
```

Comment:
For the sake of terminology consistency: all other listeners/monitors/watchers are called like FooWatcher. I would follow it here.

Reply:
Right, changed to "ResourcesWatcher".